core/trino-grammar/src/main/antlr4/io/trino/grammar/sql/SqlBase.g4

```diff
@@ -1328,6 +1329,10 @@ STRING
     : '\'' ( ~'\'' | '\'\'' )* '\''
     ;
 
+DOLLAR_STRING
```

Comment:
Why is it needed?

Reply:
This encloses the body of the function. See https://www.postgresql.org/docs/current/sql-syntax-lexical.html#SQL-SYNTAX-DOLLAR-QUOTING

Will Trino support adding text between dollars? Using `$body$` instead of just `$$` might be more readable to people not familiar with writing functions.

Reply:
Yes, it's not strictly needed, but makes it much nicer, as you don't have to deal with escaping quotes.

We could implement "tag" support if needed. I didn't do that because it's more complicated, requiring adding code to the lexer to track the tag. This shows how to do it: https://github.com/antlr/grammars-v4/blob/master/sql/postgresql/PostgreSQLLexer.g4

Reply:
Add to documentation and release notes